### PR TITLE
Itk 5.3.0

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/build-test-publish.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           python -m pip install ninja
 
       - name: Get specific version of CMake, Ninja
-        uses: lukka/get-cmake@v3.18.3
+        uses: lukka/get-cmake@v3.19.7
```
Comment (dzenanz) [inline .github/workflows/build-test-publish.yml]: Changes in this file should be squashed into the next commit.

**Review by dzenanz — APPROVED** [review]

**Comment by thewtex** [comment]
@dzenanz why is there a Windows warning:

```
D:\a\bld\ITK\Modules\ThirdParty\ZLIB\src\itkzlib-ng\crc32_braid.c(128): warning C4311: 'type cast': pointer truncation from 'const uint8_t *' to 'unsigned long'
```

**Review by dzenanz — APPROVED** [review]

**Review by dzenanz — COMMENTED** [review]
It looks like I was wrong, and external zlib wasn't the problem.

**Comment by dzenanz** [comment]
We rarely use/test `ITK_LEGACY_SILENT` option.

**Comment by dzenanz** [comment]
Maybe I was looking at the wrong build, and now the problem are [3 failing tests](https://open.cdash.org/viewTest.php?onlytimestatus&buildid=8307307)?

**Comment by dzenanz** [comment]
I issued a rerun of the failed jobs. With updated time, it will be easier to tell what is the problem.

**Comment by dzenanz** [comment]
The compile warnings are still there, `D:\a\bld\ITK\Modules\ThirdParty\ZLIB\src\itkzlib-ng\crc32_braid.c(128): warning C4311: 'type cast': pointer truncation from 'const uint8_t *' to 'unsigned long'` and `/.../bld/ITKEx-build/src/Bridge/NumPy/ConvertNumPyArrayToitkImage/ConvertNumPyArrayToitkImage.ipynb:: WARNING: Pygments lexer name 'ipython3' is not known`.

**Comment by dzenanz** [comment]
There is an outstanding PR to fix this in `zlib-ng`: https://github.com/zlib-ng/zlib-ng/pull/1371. I still don't know why it doesn't show up on our nightly builds.

File: Superbuild/External-Python.cmake [hunk 2]
```diff
@@ -17,6 +17,6 @@ ExternalProject_Add(ITKPython
   DOWNLOAD_COMMAND ""
   CONFIGURE_COMMAND ${PYTHON_EXECUTABLE} -m venv "${_itk_venv}"
   BUILD_COMMAND ${ITKPYTHON_EXECUTABLE} -m pip install --upgrade pip
-  INSTALL_COMMAND ${ITKPYTHON_EXECUTABLE} -m pip install --ignore-installed itk>=5.3.0 sphinx==4.4.0 docutils<0.18 six black nbsphinx ipython sphinx-contributors ipykernel matplotlib itkwidgets[lab,notebook]>=1.0a21 pydata-sphinx-theme ipython
+  INSTALL_COMMAND ${ITKPYTHON_EXECUTABLE} -m pip install --ignore-installed itk>=5.3.0 sphinx==4.4.0 docutils<0.18 six black nbsphinx ipywidgets sphinx-contributors ipykernel matplotlib itkwidgets[lab,notebook]>=1.0a21 pydata-sphinx-theme ipython
```
Comment (dzenanz) [inline Superbuild/External-Python.cmake:20]: I now notice that `ipython` was already on this list, in the last place.

**Pull request merged**